src/utils/types.d.ts

```diff
@@ -0,0 +1,44 @@
+export type FrameworkNames = '#static' | '#auto' | '#custom'
```

Comment:
Framework names can also be arbitrary strings

Reply:
thx updated :)